xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -141,7 +141,7 @@ void CPVRRecordings::GetSubDirectories(const CPVRRecordingsPath &recParentPath,
 
   // Remove the watched overlay from folders containing unwatched entries
   for (auto item : unwatchedFolders)
-    item->SetOverlayImage(CGUIListItem::ICON_OVERLAY_WATCHED, true);
+    item->SetOverlayImage(CGUIListItem::ICON_OVERLAY_UNWATCHED);
```

Comment:
Okay, this makes sense. May I suggest to change the code comment to make clear that we're setting unwatched here. Makes things more clear. 

And could you please add the parameter "false" to SetOverlayImage. Makes the code more clear, imo.